guides/content/release_notes/3_2_0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Spree 3.2.0
+section: version
+---
+
+## Major/New Features
+
+## Upgrade
+
+### Update Gemfile & Run Migrations
+
+### Other Gotchas
+
+#### Noteworthy Changes
+
+Also please review each of the noteworthy changes, and ensure your customizations
+or extensions are not effected. If you are affected by a change, and have any
+of your own tips please submit a PR to help the next person!
+
+## Full Changelog
+
+You can view the full changes using [Github Compare](https://github.com/spree/spree/compare/3-1-stable...3-2-stable).
```

Comment:
@damianlegawiec The comparison link is not correct as spree do not have a 3-2-stable branch. We should compare 3-1-stable branch to master. https://github.com/spree/spree/compare/3-1-stable...master